src/app/public/src/modules/footer/footer.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="stache-footer-wrapper">
+  <div class="stache-footer-content">
+    <h3 *ngIf="siteName" class="stache-footer-site-name">
+      <strong>{{ siteName }}</strong>
```

Comment:
I would remove this extraneous `strong` tag and just add styles to the `stache-footer-site-name` class.